Sources/ShortcutFoundation/Core/Payment/ApplePayAuthorizationHandler.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  ApplePayAuthorizationHandler.swift
+//  ShortcutFoundation
+//
+//  Created by Swathi on 2022-02-13.
+//  Copyright © 2022 No more yellow notes AB. All rights reserved.
```

Comment:
All the copyrights in ShortcutFoundation should be for our company. Please, adjust it.
Maybe if something missed from others, we need to adjust.

Reply:
Fixed